code/modules/economy/chip.dm

```diff
@@ -0,0 +1,200 @@
+/obj/item/chip
+	name = "0 Chip"
+	var/initial_name = "Chip"
+	desc = "It's worth 0 Chips."
+	gender = PLURAL
+	icon = 'icons/obj/items.dmi'
+	icon_state = "chip1"
+	opacity = 0
+	density = 0
+	anchored = 0.0
+	force = 1.0
+	throwforce = 1.0
+	throw_speed = 3
+	throw_range = 10
+	w_class = ITEMSIZE_SMALL
+	var/access = list()
+	var/worth = 0
```

Comment:
I'm inclined to say this should be done with types, since these chips do not actually have a worth